test/lint.js

```diff
@@ -0,0 +1,95 @@
+var fs = require('fs');
+var path = require('path');
+var Ajv = require('ajv');
+var ajv = new Ajv({ allErrors: true });
+var hasErrors = false;
+
+function jsonDiff(actual, expected) {
+  var actualLines = actual.split(/\n/);
+  var expectedLines = expected.split(/\n/);
+
+  for (var i = 0; i < actualLines.length; i++) {
+    if (actualLines[i] !== expectedLines[i]) {
+      return [
+        '#' + i + '\x1b[0m',
+        '    Actual:   ' + actualLines[i],
+        '    Expected: ' + expectedLines[i]
+      ].join('\n');
+    }
+  }
+}
+
+function checkStyle(filename) {
+  var actual = fs.readFileSync(filename, 'utf-8').trim();
+  var expected = JSON.stringify(JSON.parse(actual), null, 2);
+
+  if (actual === expected) {
+    console.log('\x1b[32m');
+    console.log('  Style – OK');
+    console.log('\x1b[0m');
```

Comment:
There is too much white space generated here for my taste. Maybe combine into one ``console.log`` like this?
```
console.log('\x1b[32m  Style – OK\x1b[0m');
```